db/migrate/20180120013036_create_budgets.rb

```diff
@@ -0,0 +1,8 @@
+class CreateBudgets < ActiveRecord::Migration[5.1]
+  def change
+    create_table :budgets do |t|
+      t.string :name, null: false
+      t.string :budget_type, default: "basic"
```

Comment:
I would consider using Enums for `budget_type`, which means line 5 would have to change to:
`t.integer, :budget_type, default: 0`

Check the curriculum for enums

Reply:
Mmm, I like this - thanks Elle - not sure yet though if I want this to be customisable by the user, ie. let the name it whatever they want in line with why they are creating it.

Reply:
If you wish to have set types of budgets, enum makes sense.